hack/release_common.sh

```diff
@@ -24,7 +24,7 @@ requireCloudProvider(){
 }
 
 authenticate() {
-  aws ecr-public get-login-password --region us-east-1 | docker login --username AWS --password-stdin ${RELEASE_REPO}
+  aws ecr-public get-login-password --region us-east-1 | docker login --username AWS --password-stdin public.ecr.aws
```

Comment:
This is a great simplification if it works!

Reply:
Also -- you can rely on ko instead of docker to reduce our dependency closure
```suggestion
  aws ecr-public get-login-password --region us-east-1 | ko login --username AWS --password-stdin public.ecr.aws
```

Reply:
Ah, didn't realize you could do this, neat suggestion 🎉